Example/Example/LogoutViewController.swift

```diff
@@ -7,29 +7,20 @@
 //
 
 import UIKit
+import Ably
 
 class LogoutViewController: UIViewController {
 
     override func viewDidLoad() {
         super.viewDidLoad()
 
-        // Do any additional setup after loading the view.
     }
 
-    override func didReceiveMemoryWarning() {
-        super.didReceiveMemoryWarning()
-        // Dispose of any resources that can be recreated.
+    override func prepareForSegue(segue: UIStoryboardSegue, sender: AnyObject!) {
+        if segue.identifier == "toLoginSegue" {
+            client.connection.close()
```

Comment:
Just to be clear as I've not run this locally, you can log out and then log back in again?